CHANGELOG.rst

```diff
@@ -7,6 +7,10 @@ Change Log
 v3.x
 ----
 
+*Added*
+* For the ALJ potential, ``shape.rounding_radii`` now defaults to 0.0.
+
+
```

Comment:
```suggestion```
Please write the suggested change log entry in the PR discussion. I update `CHANGELOG.md` in batches to avoid merge conflicts between branches.